packages/faucet/package.json

```diff
@@ -27,7 +27,7 @@
     "eth-lib": "^0.2.8",
     "firebase": "^6.2.2",
     "firebase-admin": "^7.0.0",
-    "firebase-functions": "^2.2.2",
+    "firebase-functions": "^2.3.1",
```

Comment:
Does this match what is in `packages/verification-pool-api`? We try to keep versions the same across different packages.

Reply:
Sure. Updated both packages to use version 2.3.1